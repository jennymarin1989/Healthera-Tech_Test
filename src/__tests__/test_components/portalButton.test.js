import React from 'react';
import styled from 'styled-components';
import render from 'react-test-renderer';
import PortalButton from '../../components/styling/portalButton';
import 'jest-styled-components';


describe('Portal Patient button', ()=>{
    let title = render.create(<PortalButton />).toJSON();
  it('should render the button style correctly', () =>{
     expect(title).toMatchSnapshot();
     expect(title).toHaveStyleRule('background', 'transparent', 'color', 'MediumSeaGreen','border', 'none', ) 
  })  
})