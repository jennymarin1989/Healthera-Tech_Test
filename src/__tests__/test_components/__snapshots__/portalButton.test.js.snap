// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Portal Patient button should render the button style correctly 1`] = `
.c0 {
  background: transparent;
  color: MediumSeaGreen;
  padding: 10px 20px;
  border: none;
  font-size: 0.9em;
  font-family: verdana;
}

.c0:hover {
  border: #DCDCDC;
}

<button
  className="c0"
/>
`;
